cpp/src/arrow/engine/substrait/relation_internal.cc

```diff
@@ -698,6 +725,16 @@ Result<std::unique_ptr<substrait::ReadRel>> NamedTableRelationConverter(
   return std::move(read_rel);
 }
 
+std::unique_ptr<substrait::RelCommon> GetRelCommonEmit(std::vector<int>& emit_fields) {
```

Comment:
```suggestion
std::unique_ptr<substrait::RelCommon> MakeRelCommonEmit(std::vector<int>& emit_fields) {
```

Minor nit on the naming